contributors.html

```diff
@@ -0,0 +1,297 @@
+      <li>2016-01-22 - <a href="https://github.com/pjhampton">pjhampton</a></li>
+      <li>2016-01-23 - <a href="https://github.com/jesusangelm">jesusangelm</a></li>
+      <li>2016-01-25 - <a href="https://github.com/zhyu">zhyu</a></li>
+      <li>2016-01-31 - <a href="https://github.com/iamrahulroy">iamrahulroy</a></li>
+      <li>2016-02-07 - <a href="https://github.com/tiagobarreto">tiagobarreto</a></li>
+      <li>2016-02-07 - <a href="https://github.com/roadev">roadev</a></li>
+      <li>2016-02-18 - <a href="https://github.com/joneslee85">joneslee85</a></li>
+      <li>2016-02-19 - <a href="https://github.com/brain-geek">brain-geek</a></li>
+      <li>2016-02-19 - <a href="https://github.com/vnnoder">vnnoder</a></li>
+      <li>2016-02-23 - <a href="https://github.com/pragmaticivan">pragmaticivan</a></li>
+      <li>2016-02-23 - <a href="https://github.com/michalvalasek">michalvalasek</a></li>
+      <li>2016-02-27 - <a href="https://github.com/rodrigoddalmeida">rodrigoddalmeida</a></li>
+      <li>2016-02-27 - <a href="https://github.com/brunoasantos">brunoasantos</a></li>
+      <li>2016-02-29 - <a href="https://github.com/psantos10">psantos10</a></li>
+      <li>2016-03-01 - <a href="https://github.com/m0a">m0a</a></li>
+      <li>2016-03-02 - <a href="https://github.com/timlew">timlew</a></li>
+      <li>2016-03-03 - <a href="https://github.com/inchingforward">inchingforward</a></li>
+      <li>2016-03-05 - <a href="https://github.com/jessejanderson">jessejanderson</a></li>
+      <li>2016-03-10 - <a href="https://github.com/philss">philss</a></li>
+      <li>2016-03-10 - <a href="https://github.com/ruan-brandao">ruan-brandao</a></li>
+      <li>2016-03-13 - <a href="https://github.com/derick">derick</a></li>
+      <li>2016-03-14 - <a href="https://github.com/Slotos">Slotos</a></li>
+      <li>2016-03-15 - <a href="https://github.com/Denys-Soares">DenysonSoares</a></li>
+      <li>2016-03-17 - <a href="https://github.com/Diog-eves">Diogoneves</a></li>
+      <li>2016-03-20 - <a href="https://github.com/joeltrew">joeltrew</a></li>
+      <li>2016-03-21 - <a href="https://github.com/beeetooo">beeetooo</a></li>
+      <li>2016-03-21 - <a href="https://github.com/jadercorrea">jadercorrea</a></li>
+      <li>2016-03-22 - <a href="https://github.com/victorlcampos">victorlcampos</a></li>
+      <li>2016-03-23 - <a href="https://github.com/gaynetdinov">gaynetdinov</a></li>
+      <li>2016-03-23 - <a href="https://github.com/tchoutri">tchoutri</a></li>
+      <li>2016-03-24 - <a href="https://github.com/julienXX">julienXX</a></li>
+      <li>2016-03-25 - <a href="https://github.com/Kosmas">Kosmas</a></li>
+      <li>2016-03-26 - <a href="https://github.com/ajhager">ajhager</a></li>
+      <li>2016-03-27 - <a href="https://github.com/davidvanegas7">davidvanegas7</a></li>
+      <li>2016-03-28 - <a href="https://github.com/renatosuero">renatosuero</a></li>
+      <li>2016-03-28 - <a href="https://github.com/nscycl-e">nscyclone</a></li>
```

Comment:
Something went wrong with the Github links. For example this and the next one seem to be incorrect.